readme.md

```diff
@@ -12,6 +12,21 @@ Download the script (`wget https://github.com/X1011/git-directory-deploy/raw/mas
 - **default_username**, **default_email**: identity to use for git commits if none is set already. Useful for CI servers.
 - **repo**: repository to deploy to. Must be readable and writable. The default of "origin" will not work on Travis CI, since it uses the read-only git protocol. In that case, it is recommended to store a [GitHub token](https://help.github.com/articles/creating-an-access-token-for-command-line-use) in a [secure environment variable](http://docs.travis-ci.com/user/environment-variables/#Secure-Variables) and use it in an HTTPS URL like this: <code>repo=https://$GITHUB_TOKEN@github\.com/<i>user</i>/<i>repo</i>.git</code> **Warning: there is currently [an issue](https://github.com/X1011/git-directory-deploy/issues/7) where the repo URL may be output if an operation fails.**
 
+You can also define any of variables using environment variables and configuration files:
+
+- `GIT_DEPLOY_DIR`
+- `GIT_DEPLOY_BRANCH`
+- `GIT_DEPLOY_REPO`
+
+The script will set these variables in this order of preference:
+
+1. Defaults set in the script itself.
+* Environment variables.
+* `.env` file in the path where you're running the script.
+* File specified on the command-line (see the `-c` option below).
```

Comment:
hmm, i didn't know you could use bullets for numbers in a numbered list in Markdown, and i guess it would be easier to maintain that way, but i'd prefer using numbers for all the items for readability of the plain text.